AppCenter/AppCenterTests/MSDBStorageTests.m

```diff
@@ -419,6 +419,47 @@ - (void)testDefaultDatabaseSize {
   XCTAssertEqual(self.sut.maxPageCount, expectedPageCount);
 }
 
+- (void)testNewDatabaseIsAutoVacuumed {
+
+  // Then
+  XCTAssertTrue([self autoVacuumIsSetToFull]);
+}
+
+- (void)testNonAutoVacuumingDatabaseIsManuallyVacuumedAndAutoVacuumedWhenInitialized {
+
+  // If
+
+  // Reset database and ensure that auto_vacuum is disabled.
+  [self.storageTestUtil deleteDatabase];
+  sqlite3 *db = [self.storageTestUtil openDatabase];
+  sqlite3_exec(db, "PRAGMA auto_vacuum = NONE; VACUUM", NULL, NULL, NULL);
+  sqlite3_close(db);
+
+  // When
+  self.sut = [[MSDBStorage alloc] initWithSchema:self.schema version:0 filename:kMSTestDBFileName];
+
+  // Then
+  XCTAssertTrue([self autoVacuumIsSetToFull]);
+}
+
+- (void) testDatabaseThatIsAutoVacuumedNotManuallyVacuumedWhenInitialized {
+
+  // If
+
+  // Reset database and ensure that auto_vacuum is enabled.
+  [self.storageTestUtil deleteDatabase];
+  sqlite3 *db = [self.storageTestUtil openDatabase];
+  sqlite3_exec(db, "PRAGMA auto_vacuum = FULL; VACUUM", NULL, NULL, NULL);
+  sqlite3_close(db);
+  id dbStorageMock = OCMClassMock([MSDBStorage class]);
+
+  // Then
```

Comment:
`Then` part is before `When` part

Reply:
That's because we need to call `OCMReject` (the "then" part) before the "When" part